tests/themes/test_package.py

```
@@ -0,0 +1,33 @@
+    "structure": {"dock": {"variant": "windows-taskbar"}},
+    "effects": [{"module": "crt", "params": {}}],
+    "requires": ["assistant", "launcher"],
+}
+
+def test_extract_valid_theme(tmp_path):
+    m = extract_theme_package(_zip(MANIFEST, {"assets/wall.png": "x"}), themes_root=tmp_path)
+    assert m["id"] == "matrix-terminal"
+    assert (tmp_path / "matrix-terminal" / "assets" / "wall.png").exists()
+
+def test_invalid_config_rejected(tmp_path):
+    bad = dict(MANIFEST); bad["tokens"] = {"--evil": "x"}
```

Ruff (0.15.14)

[error] Multiple statements on one line (semicolon)

(E702)